⚡️ installer - Adding a requirement check for python3-venv

nuaimat opened the pull request:
checking for python-venv if it's installed or gives the user a tip on how to install it if it's not already installed. 
this has been tested on ubuntu 21.10

nuaimat committed:
installer - Adding a requirement check for python3-venv

Rikj000 commented:
Thank you @nuaimat for your 1st PR :pray: 
But sadly I can't merge it, on my Manjaro system it throws following error:
```
 ./installer.sh: line 172: dpkg: command not found
 🙉  Python3-venv is not installed. Can't proceed. install it with: apt-get install python3.9-venv
```

I also wonder if you can't simply install `virtualenv` using `pip install virtualenv` on Ubuntu based systems? :thinking: 
If that would be the case, then we can simply add a `virtualenv` entry in the `requirements-mgm.txt` & `Pipfile(.lock)` files

Rikj000 committed:
⚡️ Check if `virtualenv` is installed through `pip3`

Rikj000 committed:
⚡️ `pip` => `pip3`

Rikj000 commented:
@nuaimat can you try these changes & installing `virtualenv` through `pip3 install virtualenv` on your Ubuntu system?

Rikj000 committed:
⚡️ `installer.sh` Switched to `Ubuntu` & `Debian` specific `Python3-venv` check

⚡️ `installer.sh` Switched to `Ubuntu` & `Debian` specific `Python3-venv` check
